1-Draft/RFCXXXX-ScriptBlockLogging.md

```diff
@@ -0,0 +1,213 @@
+---
+RFC: RFCXXXX
+Author: Travis Plunk
+Status: Draft
+SupersededBy:
+Version: 1.0
+Area: Engine
+Comments Due: 08/15/2019
+Plan to implement: Yes
+---
+
+# Title
+
+ScriptBlock Logging
+
+## Description
+
+This is an addition to the existing ScriptBlock creation logging.
+The goal is to make it easier to configure to collect the logs centrally.
```

Comment:
I think it's worth calling out why remote logging is desirable.  If the machine gets pwned, an attacker can just clear the local logs.